FastSurferCNN/README.md

```diff
@@ -131,6 +131,21 @@ python3 generate_hdf5.py \
 
 ```
 
+#### Example Command Sagittal using --data_dir instead of --csv_file
+--data_dir specifies the path in which the data is located, with --pattern we can select subjects from the specified path. By default the pattern is "*" meaning all subjects will be selected.
+In the following example we selecting subject1 till subject19.
+```
+python3 generate_hdf5.py \
+--hdf5_name ../data/training_set_cispa_axial.hdf5 \
+--data_dir ../data \
+--pattern "subject1[0-9]" \
```

Comment:
this looks like you are selecting subject10 to subject19 ? It should match the description above.